webhook_receiver_shopify/views.py

```diff
@@ -117,13 +132,22 @@ def order_delete(_, conf, data):
 @extract_webhook_data
 def order_update(_, conf, data):
     payload = data.content
+
+    is_payload_valid, error_msg = validate_webhook_payload(payload)
```

Comment:
Same here. Please change the variable name to something like `is_valid_tags`.

Reply:
Updated. Thanks, Pooja!